scripts/slack/notify_success_no_publish.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python3
+
+import requests
+import json
+
+circle_build_url = os.getenv('CIRCLE_BUILD_URL')
+url = os.getenv('SLACK_WEBHOOK')
+
+data = {
+  'attachments': 
+  [
+    {
+      'color': '#7CD197',
+      'fallback': 'Build Notification: ' + circle_build_url,
+      'title': 'Kubernetes-Monitor Publish Notification',
+      'text': ':egg_fancy: Successful `master` merge, but no `gh-pages` release occurring :egg_fancy:'
+    }
+  ]
+}
+
+requests.post(url, data = json.dumps(data))
```

Comment:
can remove the spaces around the ```=```
(in other places as well, when passed as named arguments to a function)